### DISCUSSION TITLE
Issue with "Could not execute entrypoint stage 'main' due to errors, provided by 'terrablender'!" error code upon launch

### DISCUSSION THREAD

**Discussion by mushramoo** [FabricMC/fabric-api#3063] [Player support]
I have tried to use the other solutions people mentioned about similar issues, but none of them have worked. I have been agonizing over it for hours, reinstalled fabric, terrablender, and even messed with versions and rechecked my dependencies. I will upload the crash report, but for context, it was working perfectly fine earlier today until it just decided not to when I came back to it.  This is not my computer btw, but nobody has been here since I was gone (for a whole 6 hours lol).

The report from minecraft:
[crash-2023-05-14_20.15.58-client.txt](https://github.com/FabricMC/fabric/files/11473274/crash-2023-05-14_20.15.58-client.txt)

The log from CurseForge (app):
[latest.log](https://github.com/FabricMC/fabric/files/11473277/latest.log)

I'm new to github so i apologize if i'm doing something wrong!

**Comment by warjort** [comment]
`Caused by: java.lang.IllegalArgumentException: Jankson file reading for "C:\Users\Lauren\curseforge\minecraft\Instances\Fabric Moment\config\byg\overworld\byg-overworld-biomes.json5" failed due to the following error(s):
java.lang.IllegalArgumentException: Jankson file reading for "C:\Users\Lauren\curseforge\minecraft\Instances\Fabric Moment\config\byg\overworld\byg-overworld-biomes.json5" failed due to the following error(s):
No key global_biome_swapper in MapLike[{ /* Global toggle to enable or disable BYG's overworld biomes. */ "overworld_enabled": true, /* A list of weighted regions containing a unique biome layout. */ /* Regions may be inlined or may call a file from "this_file_parent_directory/regions" */ "regions": [ "region_1", "region_2", "region_3", "rare_region_1" ] }]`

Your config/byg/overworld/byg-overworld-biomes.json5 configuration file is broken.
Ask  the byg mod author how to fix it.
But usually you can delete configuration files and then they will be recreated with default values.

**Comment by mushramoo** [comment]
Thank you so much!!!!!!
